saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -1079,6 +1135,11 @@ msgstr ""
 msgid "The request has been upated and the approver will be notified."
 msgstr "La demande a été mise à jour et l'approbateur en sera informé."
 
+#: templates/request/saas_request.html:11
+#: templates/request/view_request.html:11
+msgid "Software as a Service Request Form"
+msgstr "Logiciel en tant que formulaire de demande de service"
```

Comment:
I think the syntax got scrambled:
"Logiciel en tant que service" should be together.